.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_YOUTUBE_PLAYLIST_ITEMS_API = "https://youtube.googleapis.com/youtube/v3/playlistItems"
+REACT_APP_YOUTUBE_CHANNELS_LIST_API = "https://www.googleapis.com/youtube/v3/channels"
+REACT_APP_YOUTUBE_SEARCH = "https://content-youtube.googleapis.com/youtube/v3/search"
+REACT_APP_YOUTUBE_API_KEY="AIzaSyAx6keB9STIcO-bkIKuy-ubU6CZO9pnwXQ"
```

Comment:
I suggest you don't commit sensitive data, you can add a .env.example with placeholder values and add the .env extension to the .gitignore